docs/src/config.ts

```diff
@@ -70,4 +70,16 @@ export const sidebar = [
 
 export const site = {
   title: 'Astro Documentation',
```

Comment:
As a pattern for others to follow, what are your thoughts on having all of these be exports?

```
export const SITE_TITLE = 'Astro Documentation';
export const SITE_DESCRIPTION = 'Build faster websites with less client-side Javascript';
```

Also, as a pattern, should this file be `site.ts` instead of `config.ts`?

Neither of these are very pressing, just thoughts that jumped out as I read through this.

Reply:
I set it up this way for the goal of scoping the variables in a way that makes sense, so you can have `site.twitter.site` instead of SITE_TWITTER_SITE or something similar